service/controller/clusterapi/v28/resource/machinedeploymentsubnet/create.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/giantswarm/operatorkit/controller/context/reconciliationcanceledcontext"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/giantswarm/aws-operator/pkg/label"
+	"github.com/giantswarm/aws-operator/pkg/annotation"
```

Comment:
This was a tough call. Label would have enabled querying of resource based on subnet, but K8s label value restrictions don't allow `/` and therefore it would have required ugly hacks that one must remember during debugging so we decided that it's better to have it as-is in the correct form in CR than perform annoying encoding/decoding of special characters (or using split fields) with label.